pennylane/ops/qubit.py

```diff
@@ -1048,6 +1048,14 @@ class CRX(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
+
+    # gradient recipe
+    c1 = (np.sqrt(2) - 4 * np.cos(np.pi / 8)) / (4 - np.cos(np.pi / 8))
+    c2 = (np.sqrt(2) - 1) / (4 * np.cos(np.pi / 8) - 2)
+    a = np.pi / 2
+    b = 3 * np.pi / 4
```

Comment:
Does it look similar for `CRY`, `CRZ`? We could make these constants to save calculation each time (although the overhead is probably relatively small).

Reply:
These are class constants, so probably no need to worry 🙂 They will be computed on import, not on operation instantiation.